Model/Service/DumpCmsDataService.php

```diff
@@ -76,6 +80,24 @@ private function write(WriteInterface $writeDirectory, string $filePath, string
         $stream->close();
     }
 
+    private function getStoreCodes($stores): array
+    {
+        $storeCodes = [];
+        if ($stores) {
+            foreach ($stores as $storeId) {
+                try {
+                    $store = $this->storeManager->getStore($storeId);
+                    $storeCode = $storeId ? $store->getCode() : self::STORE_SCOPE_ALL;
```

Comment:
if $storeId = 0, then we already have all store scopes (admin scope), in that case it should be:
`return [self::STORE_SCOPE_ALL];`
since nothing else is needed (you can't select ALL + additional store scopes).